frontend/javascripts/dashboard/dataset/dataset_settings_viewconfig_tab.tsx

```diff
@@ -50,23 +115,45 @@ export default function DatasetSettingsViewConfigTab() {
       dataIndex: "comment",
     },
   ];
-  const comments: Partial<Record<keyof DatasetLayerConfiguration, string>> = {
-    alpha: "20 for segmentation layer",
-    min: "Only for color layers",
-    max: "Only for color layers",
-    intensityRange: "Only for color layers",
+  const comments: Partial<Record<keyof DatasetLayerConfiguration, [string, string | null]>> = {
+    alpha: ["20 for segmentation layer", null],
+    min: [
+      "Only for color layers",
+      "The minimum possible color range value adjustable with the histogram slider.",
+    ],
+    max: [
+      "Only for color layers",
+      "The maximum possible color range value adjustable with the histogram slider.",
+    ],
+    intensityRange: [
+      "Only for color layers",
+      "The color value range between which color values are interpolated and shown.",
+    ],
+    mapping: [
+      "Active Mapping",
+      "The mapping whose type and name is active by default. This field is an object with the keys 'type' and 'name' like {name: 'agglomerate_65', type: 'HDF5'}.",
+    ],
   };
   const layerViewConfigurationEntries = _.map(
     { ...getDefaultLayerViewConfiguration(), min: 0, max: 255, intensityRange: [0, 255] },
     (defaultValue: any, key: string) => {
       // @ts-ignore Typescript doesn't infer that key will be of type keyof DatasetLayerConfiguration
       const layerViewConfigurationKey: keyof DatasetLayerConfiguration = key;
       const name = layerViewConfigurations[layerViewConfigurationKey];
+      const commentParts = comments[layerViewConfigurationKey];
+      const comment =
+        commentParts != null && commentParts[1] != null ? (
```

Comment:
Using named keys would make reading this easier. If you add a description for `alpha` then there is no need to check them for `null`.